.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "java.compile.nullAnalysis.mode": "disabled"
+}
```

Comment:
Is this committed by mistake or necessary?

Reply:
reverted back